packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/utils.ts

```diff
@@ -48,6 +49,21 @@ export const buildSelectedRadioButtons = <T extends CheckboxOption>(
 	}, {});
 };
 
+export const buildDisabledToggleButtons = (
+	buttons: IToggleButton[],
+	disabled: boolean
+): { [key: string]: boolean } => {
+	const disabledButtons:{[key: string]: boolean} = {}
+
+	buttons.forEach(button => {
+		disabledButtons[button.value] = button.disabled ?? false;
+	});
+
+	disabledButtons[ALL_BUTTON_VALUE] = disabled || Object.values(disabledButtons).some(value => value);
```

Comment:
the disabled flag that was passed initially should only be used here?

Reply:
I updated this util so that here we build the buttons and the disabled prop is passed directly to the `kv-toggle-group` component, so if its value is `true` it will override the config